example/build.yaml

```diff
@@ -0,0 +1,30 @@
+# Read about `build.yaml` at https://pub.dartlang.org/packages/build_config
+builders:
+  # name of the builder
+  member_count:
+    # library URI containing the builder - maps to `lib/member_count_library_generator.dart`
+    import: "package:source_gen_example/builder.dart"
+    # Name of the function in the above library to call.
+    builder_factories: ["metadataLibraryBuilder"]
+    # The mapping from the source extension to the generated file extension
+    build_extensions: {".dart": [".info.dart"]}
+    # Will automatically run on any package that depends on it
```

Comment:
[nit] Just as a note, this will only run on the _root_ package if it depends on it because of the `build_to: source`. I'm not sure if that's worth calling out here or not.

Reply:
I thought that was implied. No?

Reply:
🤷‍♂️ 

I guess it's implied, but as I read this I was thrown off by the "any package" here.